proto/pocket/migration/morse_offchain.proto

```diff
@@ -133,6 +137,9 @@ message MorseValidator {
   // The string representation of the BigInt amount of upokt.
   string staked_tokens = 7 [(gogoproto.jsontag) = "tokens"];
 
+  // TODO_IN_THIS_COMMIT: comment...
+  google.protobuf.Timestamp UnstakingCompletionTime = 8 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true, (gogoproto.jsontag) = "unstaking_time", (gogoproto.moretags) = "yaml:\"unstaking_time\""];
```

Comment:
Can we call it `UnbondingTimestamp`?

Reply:
This is copy/paste from Morse. If we change it to anything, I would argue that it should be to match the JSON tag, **which we CANNOT change** since that's what it will be called in the JSON that's output from the Morse CLI.

```suggestion
  google.protobuf.Timestamp unstaking_time = 8 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true, (gogoproto.jsontag) = "unstaking_time", (gogoproto.moretags) = "yaml:\"unstaking_time\""];
```